pkg/porter/logs.go

```diff
@@ -0,0 +1,66 @@
+package porter
+
+import (
+	"fmt"
+
+	"get.porter.sh/porter/pkg/context"
+	"github.com/cnabio/cnab-go/claim"
+	"github.com/pkg/errors"
+)
+
+// LogsShowOptions represent options for an installation logs show command
+type LogsShowOptions struct {
+	sharedOptions
+	ClaimID string
+}
+
+// Installation name passed to the command.
+func (o *LogsShowOptions) Installation() string {
+	return o.sharedOptions.Name
+}
+
+// Validate validates the provided args, using the provided context,
+// setting attributes of OutputShowOptions as applicable
+func (o *LogsShowOptions) Validate(cxt *context.Context) error {
+	// Attempt to derive installation name from context
+	err := o.sharedOptions.defaultBundleFiles(cxt)
+	if err != nil {
+		return err
+	}
+
+	if o.File == "" && o.Name == "" && o.ClaimID == "" {
+		return errors.New("--installation or --run is required")
```

Comment:
What is the intended behavior when both `--installation` and `--run` is supplied?  (The run may be from a different installation, etc.)

Reply:
Ooh good question, I would say that they are mutually exclusive. I'll add a check for that.